src/python/WMCore/JobSplitting/JobFactory.py

```diff
@@ -200,9 +202,24 @@ def commit(self):
 
                     job['possiblePSN'] = locSet
                     if len(job['possiblePSN']) == 0:
+                        # NOTE: If we have no place to execute a single job we mark it as failedOnCreation.
+                        #       We have two options:
+                        #       * Either propagate the job state to 'createdfailed' here immediately
+                        #       * Or we let JobCreatorPoller later catch the marker and do it for us
+                        #         when moving the jobs from state 'new' to 'created'
+                        #       For the time being we opt for the later
+                        job["failedOnCreation"] = True
+                        job["failedErrCode"] = 71101
+                        job["failedReason"] = WM_JOB_ERROR_CODES[job["failedErrCode"]]
+                        msg = "Found bad job: %s from JobGroup: %s and task: %s."
```

Comment:
Please rephrase the "Found bad job" to something more meaningful, e.g.: "Found job with an empty list of possiblePSN".

Reply:
done